<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRequirementDialog</class>
 <widget class="QDialog" name="EditRequirementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>224</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Редактировать свойства требования</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
    <normaloff>:/images/raznoe31/raznoe31/48x48/42.png</normaloff>:/images/raznoe31/raznoe31/48x48/42.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="requirementComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>UR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NFR</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тип требования:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="componentBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Компонент </string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Название требования:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../klever/core/frameWork/resources/framework_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditRequirementDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditRequirementDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
